apps/user-service/.env.example

```
@@ -0,0 +1,14 @@
+# database
+DB_HOST="localhost"
+DB_PORT="5432"
+DB_USER="postgres"
+DB_PASSWORD="postgres"
+DB_NAME="task-management"
+
+FRONTEND_URL="http://localhost:3000"
+
+# mail
+MAIL_HOST="smtp.ethereal.email"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)